.packit.yaml

```diff
@@ -44,6 +44,39 @@ jobs:
   # TEST JOBS
   ## Tests on pull request stage. Tests are run on demand
   ### Definitions of the tier0 tests (non-destructive and destructive separately)
+  - &tests-stream-8
```

Comment:
Is this gonna be triggered when someone comments out "packit/test --labels tier0-sanity"?

Reply:
No it won't.
It will either come with `/packit test` and the whole suite or with `/packit test --labels stream-8`

https://github.com/danmyway/convert2rhel/blob/9741b7986a7ea63e7403f6e43be0feaf97f575d0/.packit.yaml#L77-L78